lib/src/http/cookies.rs

```diff
@@ -231,23 +231,70 @@ impl<'a> Cookies<'a> {
     /// ```
     pub fn add_private(&mut self, mut cookie: Cookie<'static>) {
         if let Cookies::Jarred(ref mut jar, key) = *self {
-            if cookie.path().is_none() {
-                cookie.set_path("/");
-            }
+            Cookies::set_cookie_defaults(&mut cookie);
 
-            if cookie.http_only().is_none() {
-                cookie.set_http_only(true);
-            }
+            jar.private(key).add(cookie)
+        }
+    }
 
-            if cookie.expires().is_none() {
-                cookie.set_expires(::time::now() + ::time::Duration::weeks(1));
-            }
+    /// Adds original `cookie` to the collection. The cookie's value is encrypted with
+    /// authenticated encryption assuring confidentiality, integrity, and
+    /// authenticity. The cookie can later be retrieved using
+    /// [`get_private`](#method.get_private) and removed using
+    /// [`remove_private`](#method.remove_private).
+    ///
+    /// Unless a value is supplied for the given key, the following defaults are
+    /// set on `cookie` before being added to `self`:
+    ///
+    ///    * `path`: `"/"`
+    ///    * `SameSite`: `Strict`
+    ///    * `HttpOnly`: `true`
+    ///    * `Expires`: 1 week from now
+    ///
+    /// These defaults ensure maximum usability and security. For additional
+    /// security, you may wish to set the `secure` flag.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use rocket::http::{Cookie, Cookies};
+    ///
+    /// fn handler(mut cookies: Cookies) {
+    ///     cookies.add_original_private(Cookie::new("name", "value"));
+    /// }
+    /// ```
+    pub fn add_original_private(&mut self, mut cookie: Cookie<'static>) {
```

Comment:
This should not be exposed to the public. This should be `pub(crate)`. Subsequently, we don't need tot he elaborate docstring. Just a simple something like:

> Adds an original, private `cookie` to this collection.